src/Settings/components/translation.jsx

```diff
@@ -0,0 +1,88 @@
+import { ListItem, Avatar, Switch } from "@rneui/themed";
+import colors from "../../common/colors";
+import STRINGS from "../../common/localization";
+import {
+  toggleEnglishTranslation,
+  togglePunjabiTranslation,
+  toggleSpanishTranslation,
+} from "../../common/actions";
+import { useSelector } from "react-redux";
+import { useState } from "react";
```

Comment:
As mentioned above too, move third party imports before the internal imports

Reply:
fixed